controllers/users/users.js

```diff
@@ -0,0 +1,61 @@
+const { validationResult } = require("express-validator/check");
+const bcrypt = require("bcryptjs");
+const jwt = require("jsonwebtoken");
+const config = require("config");
+const User = require("../../models/User");
+
+exports.registerUser = (req, res) => {
+  const errors = validationResult(req);
+
+  if (!errors.isEmpty()) {
+    return res
+      .status(400)
+      .json({ errors: errors.array({ onlyFirstError: true }) });
+  }
+
+  const { firstName, lastName, email, password } = req.body;
+
+  User.findOne({ email: email.toLowerCase() }).then(user => {
```

Comment:
I'm afraid of finding users by toLowerCase email.
What if user create account with uppercase letters? There was no replacing to lowercase new email in your method.
'EXAMPLE@gmail.com' (DB) != 'example@gmail.com' (FINDING USER)

So theoriticaly user can create many of account with different letters lower or upper

Correct me if I'm wrong

Reply:
You're right.